hat/app/org/hatdex/hat/api/controllers/Files.scala

```diff
@@ -278,33 +261,27 @@ class Files @Inject() (
         ManageFiles(file.source)
       )
     ) ||
-    WithRole.isAuthorized(user, authenticator, Owner()) ||
-    (file.permissions.isDefined &&
-    file.permissions.get.exists(_.userId == user.userId))
-  }
+      WithRole.isAuthorized(user, authenticator, Owner()) ||
+      (file.permissions.isDefined &&
+        file.permissions.get.exists(_.userId == user.userId))
 
   private def filePermissionsCleaned(
       file: ApiHatFile
     )(implicit user: HatUser,
-      authenticator: HatApiAuthEnvironment#A
-    ): ApiHatFile = {
-    if (WithRole.isAuthorized(user, authenticator, Owner())) {
+      authenticator: HatApiAuthEnvironment#A): ApiHatFile =
+    if (WithRole.isAuthorized(user, authenticator, Owner()))
       file
-    } else {
```

Comment:
Non blocking: i thought our styling rules where to include brackets? Not related to this PR but we should confirm whether we want or dont want brackets and ensure the tools are applying the correct rules

Reply:
Let me verify the scalafmt rules that my pre-commit hook is using.

Reply:
ive since noticed it other places, its ok